### PR TITLE
Logged user info

### PR DESCRIPTION
**Pull request by rabinovichm**
Trello card: https://trello.com/c/qU1HCL0C/28-acceder-a-la-informaci%C3%B3n-del-user-logueado

### CODE REVIEW

File: training/src/main/java/wolox/training/controllers/UserController.java [hunk 1]
```diff
@@ -40,6 +43,16 @@
     @Autowired
     PasswordEncoder passwordEncoder;
 
+    @GetMapping("/username")
+    public User currentUserName(Authentication authentication) throws UserNotFoundException {
+        User user = userRepository.findFirstByUsername(authentication.getName());
+
+        if(user==null){
```
Comment (Jimenarosello) [inline training/src/main/java/wolox/training/controllers/UserController.java]: This validation shouldn't be necessary, if a user is logged in then it must exist in the repository :) 
What you could validate is what happens when the user is not logged in - would "authentication" be null? If that's so, the clause authentication.getName() will throw NullPointerException.

Comment (rabinovichm) [reply training/src/main/java/wolox/training/controllers/UserController.java]: Deleted that validation. If the user is not logged in, it throws unauthorized directly (made a test for that :smile:).

Comment (Jimenarosello) [reply training/src/main/java/wolox/training/controllers/UserController.java]: Awesome! Nice job 😊

**Review by Jimenarosello — APPROVED** [review]
Nice job :)

**Pull request merged**